docs/ios/deployment/overview.md

```diff
@@ -0,0 +1,193 @@
+
+    :::image type="content" source="media/overview/account-type.png" alt-text="Xcode select the type of account you'd like to add popup.":::
+
+1. In the sign in popup, enter your Apple ID and click the **Next** button.
+1. In the sign in popup, enter your Apple ID password and click the **Next** button:
+
+    :::image type="content" source="media/overview/sign-in.png" alt-text="Xcode Apple account sign-in.":::
+
+1. In the **Accounts** tab, click the **Manage Certificates...** button to ensure that your distribution certificate has been downloaded.
+1. In the **Accounts** tab, click the **Download Manual Profiles** button to download your provisioning profiles:
+
+    :::image type="content" source="media/overview/account-details.png" alt-text="Xcode Apple Developer Program account details.":::
+
+1. Close Xcode.
+
+## Add entitlements to your app
+
+In iOS, apps run in a sandbox that provides a set of rules that limit access between the app and system resources or user data. *Entitlements* are used to request the expansion of the sandbox to give your app additional capabilities. Any entitlements used by your app must be specified in an entitlements file. For more information about entitlements, see [Entitlements](entitlements.md).
+
+## Add code signing data to your app project
+
+There are project-level settings you must set to sign your iOS app with its provisioning profile. These settings are configured in a `<PropertyGroup>` node:
+
+- `<RuntimeIdentifier>` – the runtime identifier (RID) for the project. Set to `ios-arm64`.
+- `<CodesignKey>` – the name of the distribution certificate you installed into Keychain Access on your Mac build host.
+- `<CodesignProvision>` – the provisioning profile name.
+- `<CodesignEntitlement>` – the name of the entitlements file. Set to `Entitlements.plist`. This setting need only be specified if you're using entitlements.
+- `<ArchiveOnBuild>` – a boolean value that indicates whether to build the app package. Set to `true`.
+- `<TcpPort>` – the TCP port on which to communicate with your Mac build host. Set to `58181`.
+- `<ServerAddress>` – the IP address of your Mac build host.
+- `<ServerUser>` – the username to use when logging into your Mac build host. Use your system username rather than your full name.
+- `<ServerPassword>` – the password for the username used to log into the Mac build host.
+- `<_DotNetRootRemoteDirectory>` – the folder on the Mac build host that contains the .NET SDK. Set to `/Users/{macOS username}/Library/Caches/Xamarin/XMA/SDKs/dotnet/`.
+
+For security reasons, you might not want to supply a value for all these settings in the project file. You can also provide these values on the command line when you publish the app. For example, values for `<ServerAddress>`, `<ServerUser>`, `<ServerPassword>`, and `<_DotNetRootRemoteDirectory>` will typically be provided on the command line. An example of this can be found in the [Publish](#publish) section.
```

Comment:
Would it make sense to make very clear that you can mix-and-match these methods? You can specify one thing in the csproj and the other in the command-line if that's what you want?